### DISCUSSION TITLE
Make AM sensitivity adjustable

### DISCUSSION THREAD

**Discussion by ohmytime** [PE5PVB/TEF6686_ESP32#382] [Ideas]
How about changing the sensitivity to a user-adjustable parameter in the menu
Because some user think it's too high or low.

Several sets can be preset in the menu for users to choose.

According to this commit: https://github.com/PE5PVB/TEF6686_ESP32/commit/4d7b9f703fbe266e721fcb7f455250259fd23e51

**Comment by ohmytime** [comment]
And the FM sensitivity

**Comment by Overland-DX** [comment]
I completely agree that it would have been a nice option in the menu.
I have tested slightly different values from 200 to 1200 and found that the radio often stops on weak stations and noise on values 700 ++.
But I have noticed that the radio skips many strong stations even if the sensitivity is set high.
I tested with a station that was strong in the high peeks and medium in the weakest peeks, but I had to scan over 7-8 times before it stopped on the relevant frequency.

Could there be a connection in the scan speed? Because it may seem that it does not register all the frequencies it scans.

**Comment by PE5PVB** [comment]
Done. Scan sensitivity is added to the menu.

**Comment by ohmytime** [comment]
Thank you
